src/Book.java

```diff
@@ -0,0 +1,25 @@
+public class Book {
+    private final String bookName;
+    private final Author authorName;
```

Comment:
Тоже по неймингу, у книги есть свойство "автор", а не "имя автора"
```suggestion
    private final Author author;
```